opsmngr/users.go

```diff
@@ -0,0 +1,245 @@
+// Copyright 2019 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package opsmngr
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	usersBasePath = "users"
+	projectUsersBasePath = "groups/%s/users"
+	orgUsersBasePath = "orgs/%s/users"
+)
+
+// UsersService provides access to the user related functions in the Ops Manager API.
+//
+// See more: https://docs.opsmanager.mongodb.com/current/reference/api/groups/
+type UsersService interface {
+	ListProjectUsers(context.Context, string, *atlas.ListOptions) ([]*User, *atlas.Response, error)
```

Comment:
[nit] the way the client is organised matches our docs, so this method should belong to `ProjectsService`